src/Network/Network/ExpressRouteCircuit/NewAzureExpressRouteCircuitCommand.cs

```diff
@@ -104,6 +104,12 @@ public class NewAzureExpressRouteCircuitCommand : ExpressRouteCircuitBaseCmdlet
              ValueFromPipelineByPropertyName = true)]
         public double BandwidthInGbps { get; set; }
 
+        [Parameter(
+             ParameterSetName = "ExpressRoutePort",
+             Mandatory = false,
+             ValueFromPipelineByPropertyName = true)]
+        public string AuthorizationKey { get; set; }
```

Comment:
I‘m wondering whether this value should be wrapped with [securestring](https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/design-guidelines/parameter-best-practices.md#secret-parameters).

Reply:
It's not exactly a secret as it cannot be used any further once consumed. We have other resources using it as well and are not using secure string for this property.